api/core/model_runtime/model_providers/_position.yaml

```diff
@@ -23,6 +23,7 @@
 - tongyi
 - wenxin
 - moonshot
+- tencent
```

Comment:
tencent_cloud might be better?

Reply:
other module like tencent cos also use 'tencent', I think it's fine